docs/desktop-wpf/migration/convert-project-from-net-framework.md

```diff
@@ -1,11 +1,326 @@
+
+**WCF clients generated by Svcutil are meant for use on .NET Framework.** 
+
+Solutions that use svcutil-generated WCF clients will need to regenerate .NET Standard-compatible clients for use with .NET Core. One of the main reasons the old clients won't work is that they depend on app configuration for defining WCF bindings and endpoints. Because .NET Standard WCF APIs can work cross-platform (where System.Configuration APIs aren't available), WCF clients for .NET Core and .NET Standard scenarios must define bindings and endpoints programmatically instead of in configuration.
+
+In fact, any WCF client usage that depends on the `<system.serviceModel>` app.config section (whether created with Svcutil or manually) will need changed to work on .NET Core.
+
+There are two ways to automatically generate .NET Standard-compatible WCF clients:
+
+* The [dotnet-svcutil] tool is a .NET Core CLI tool that generates WCF clients similar to how Svcutil worked previously.
+* Visual Studio can generate WCF clients using the [WCF Web Service Reference][wcf-connected-service] option of its Connected Services feature.
+
+Either approach works well. Alternatively, of course, you could write the WCF client code yourself. For this sample, I chose to use the Visual Studio Connected Service feature. To do that, right-click on the *BeanTraderClient.Core* project in Visual Studio's solution explorer and select Add -> Connected Service. Next, choose the WCF Web Service Reference Provider. This will bring up a dialog where you can specify the address of the backend Bean Trader web service (`localhost:8080` if you are running the server locally) and the namespace that generated types should use (**BeanTrader.Service**, for example).
+
+![WCF Web Service Reference Connected Service Dialog](media/convert-project-from-net-framework/connected-service-dialog.png)
+
+After clicking the Finish button, a new 'Connected Services' node is added to the project and a Reference.cs file is added under that node containing the new .NET Standard WCF client for accessing the Bean Trader service. If you look at the `GetEndpointAddress` or `GetBindingForEndpoint` methods in that file, you'll see that bindings and endpoints are now generated programmatically (instead of via app config). The 'Add Connected Services' feature may also add references to some System.ServiceModel packages in the project file, which aren't needed since all necessary WCF packages are included via Microsoft.Windows.Compatibility. Check the csproj and if any extra System.ServiceModel `<PackageRefernce>` items have been added, remove them.
```

Comment:
```suggestion
After clicking the Finish button, a new 'Connected Services' node is added to the project and a Reference.cs file is added under that node containing the new .NET Standard WCF client for accessing the Bean Trader service. If you look at the `GetEndpointAddress` or `GetBindingForEndpoint` methods in that file, you'll see that bindings and endpoints are now generated programmatically (instead of via app config). The 'Add Connected Services' feature may also add references to some System.ServiceModel packages in the project file, which aren't needed since all necessary WCF packages are included via Microsoft.Windows.Compatibility. Check the csproj and if any extra System.ServiceModel `<PackageReference>` items have been added, remove them.
```